registry/registry.go

```diff
@@ -219,6 +220,10 @@ func ContinueOnError(err error) bool {
 		return shouldV2Fallback(v)
 	case *client.UnexpectedHTTPResponseError:
 		return true
+	case error:
+		if val := strings.Contains(err.Error(), errNoSpaceOnDevice.Error()); val {
```

Comment:
Ah no, this wouldn't be easy.
Instead we can do `strings.Contains(err.Error(), syscall.ENOSPC.Error())`